### PR TITLE
#5 added login handler made it functional sending post request to the…

### PR DESCRIPTION
**Pull request by yoosefkh**
issue number #70
don't merge this before  PR number #61  please ! 
Thank you !! :)
added login handler made it functional sending post request to the server !!

### CODE REVIEW

**Comment by yoosefkh** [comment]
added login handler made it functional sending post request to the server !

**Comment by shiryz** [comment]
@tawfiknasser merge conflicts

File: client/src/components/LoginComponent/login.js [hunk 1]
```diff
@@ -1,24 +1,51 @@
...
+  handleSubmit(event) {
+    event.preventDefault();
+    const { username, password } = event.target.elements;
+    const postData = {
+      username: username.value,
+      password: password.value
+    };
+    axios
+      .post("/login", postData)
+      .then(res => console.log(res))
```
Comment (shiryz) [inline client/src/components/LoginComponent/login.js]: Is this going to be changed to something else later? logging will not be useful in the future

File: server/routes/index.js [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
+router.get("/", (req, res) => {
+  res.send("helloww");
+});
+router.get("/voters", (req, res) => {
 
-router.get('/voters', (req, res) => {
   res.json(voters);
 });
+router.post("/login", (req, res) => {
+  console.log(req.body);
```
Comment (shiryz) [inline server/routes/index.js]: I'm assuming the user will be authenticated at some point here and this is not final, if so please change the log into a comment that'll you'll add authentication

**Comment by MynahMarie** [comment]
@yoosefkh Please update the description so that it is readable and linked to an issue.

File: client/src/components/LoginComponent/login.js (hunk 1 above)
Comment (yoosefkh) [reply client/src/components/LoginComponent/login.js]: sure it will .. just used  it for checking my code ! :)

File: server/routes/index.js (hunk 2 above)
Comment (yoosefkh) [reply server/routes/index.js]: changed already, will be when  i push the next PR  ^_^

File: client/src/components/LoginComponent/login.js [hunk 3]
```diff
@@ -1,24 +1,51 @@
...
+            />
+            <input
+              class="password"
+              name="password"
+              placeholder="password"
+              requierd
+            />
+          </div>
+          <div>
+            <button type="submit" id="loginbutt">
```
Comment (Karyum) [inline client/src/components/LoginComponent/login.js]: think about this id name btw _hehe_

Comment (tawfiknasser) [reply client/src/components/LoginComponent/login.js]: :laughing:

**Pull request merged**